Animal.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+class Animal
+{
+    private:
+        string Nume;
+        string Specie;
+        string Rasa;
+        string CategorieDeProductie;
+        int Greutate;
+        int Varsta;
```

Comment:
In general, este un good practice sa pui intai chestiile care sunt cu `public`, apoi `protected` si abia apoi `private`. In ideea in care eu ca si consumator al clasei tale, cand intru sa ma uit in cod, vreau intai si intai sa vad ce pot face cu codul, gen ce functii imi expune. chestiile `private` sunt "detalii de implementare"